tests/e2e/vsan_test.go

```diff
@@ -57,6 +57,10 @@ func (s *VsanTestSuite) SetUpSuite(c *C) {
 	}
 }
 
+func (s *VsanTestSuite) SetUpTest(c *C) {
+	s.policyList = []string{}
```

Comment:
Minor: As this is cleaning the list, this step should go to teardown.
You can empty this slice as s.policyList = s.policyList[:0]